language-server/test/dotty/tools/languageserver/CompletionTest.scala

```diff
@@ -1211,38 +1213,62 @@ class CompletionTest {
       ("mapExtensionMethod", Method, "[B](f: Int => B): Foo[B]"),
     )
     code"""trait Foo[A] {
-        |  def mapFoo[B](f: A => B): Foo[B] = ???
-        |}
-        |extension[A] (x: Foo[A]) {
-        |  def mapExtensionMethod[B](f: A => B): Foo[B] = ???
-        |}
-        |case class Baz[F[_]](baz: F[Int])
-        |type M[T] = T match {
-        |  case Int => Foo[Int]
-        |}
-        |case class Bar[F[_]](bar: F[Int])
-        |def foo(x: Bar[M]) = x.bar.ma${m1}"""
-          .withSource.completion(m1, expected)
+          |  def mapFoo[B](f: A => B): Foo[B] = ???
+          |}
+          |extension[A] (x: Foo[A]) {
+          |  def mapExtensionMethod[B](f: A => B): Foo[B] = ???
+          |}
+          |case class Baz[F[_]](baz: F[Int])
+          |type M[T] = T match {
+          |  case Int => Foo[Int]
+          |}
+          |case class Bar[F[_]](bar: F[Int])
+          |def foo(x: Bar[M]) = x.bar.ma${m1}""".withSource
+      .completion(m1, expected)
   }
 
   @Test def packageCompletionsOutsideImport: Unit = {
     val expected = Set(
       ("java", Module, "java"),
       ("javax", Module, "javax"),
     )
-    code"""object Foo { ja${m1}"""
-             .withSource.completion(m1, expected)
+    code"""object Foo { ja${m1}""".withSource
+      .completion(m1, expected)
   }
 
   @Test def topLevelPackagesCompletionsOutsideImport: Unit = {
-    val expected = Set(
-      ("example", Module, "example"),
-    )
     code"""package example:
           |    def foo = ""
           |
-          |def main = exa${m1}"""
-             .withSource.completion(m1, expected)
+          |def main = exa${m1}""".withSource
+      .completion(m1, Set(("example", Module, "example")))
+  }
+
+  @Test def typeAliasCompletions: Unit = {
```

Comment:
Tests below that checks the issue of missing type alias completion. Changes above are style changes to unify them in this file.